app/presenters/event_spread_display.rb

```diff
@@ -23,18 +23,14 @@ def display_style_hash
     {elapsed: "Elapsed", ampm: "AM/PM", military: "24-Hour", segment: "Segment"}
   end
 
-  def effort_times_rows
-    @effort_times_rows ||=
-      filtered_ranked_efforts.map do |effort|
-        EffortTimesRow.new(effort: effort,
-                           lap_splits: lap_splits,
-                           split_times: split_times_by_effort.fetch(effort.id, []),
-                           display_style: display_style)
-      end
-  end
-
-  def effort_times_row_ids
-    effort_times_rows.map(&:id)
+  def grouped_effort_times_rows
+    result = effort_times_rows.group_by(&:effort_status)
+    if event.laps_required == 0 && result["In Progress"].any? then
+      { "Started" => result.slice("Finished", "In Progress", "Dropped").values.flatten,
```

Comment:
This is a creative solution, but it does result in all "In Progress" runners being listed later than the "Finished" runners, even when the "In Progress" runners have completed more laps. I'd like to find a way to show this without changing runner order. 

For unlimited lap events, what do you think about grouping by `started?` instead, and then modifying the hash keys to "Started" and "Not Started"?

Reply:
Ah yea, I think I figured out a solution in my recent commits. Basically I just sort these in progress, unlimited lap events by the number of completed laps. Do I need to sort within that, more granularly, by lowest time of completion for that given number of laps?

Reply:
We've got the `overall_performance` field that incorporates everything for us, so I think we can just use that and group by whether the efforts are `started?`